src/styles/global.css

```
@@ -1787,6 +1787,45 @@ body.blog-page {
+   wrapper — it still applies to each shot via `flex-basis`. */
+.project-screenshot--pair .project-screenshot__inner {
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: center;
+  gap: var(--screenshot-pair-gap);
+}
+
+.project-screenshot--narrow.project-screenshot--pair .project-screenshot__inner,
+.project-screenshot--narrow.project-screenshot--pair .project-stack {
+  --screenshot-pair-gap: clamp(0.6rem, 1.8vw, 1.4rem);
+  max-width: calc(320px * 2 + var(--screenshot-pair-gap));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)